webclient/app/src/app/features/apps/entities/entityDiagram/EntityDiagram.jsx

```diff
@@ -94,14 +98,14 @@ function EntityDiagram(props) {
             return (
                 <Draggable
                     axis={"both"}
-                    onStop={(update, draggableData) => updatePosition(draggableData, entity)}
+                    onStop={(update, draggableData) => savePosition(entity, index, draggableData)}
                     key={entity.id + index + ""}
                     defaultClassName={entityDiagramStyles.draggable}
                     position={entityCardPosition}
                     disabled={!editable}
                 >
                     <div>
-                        <EntityCard entity={entity} handleEdit={setCurrentEntity} handleDelete={deleteEntity}
+                        <EntityCard entity={entity} onSelect={setSelectedEntityId} handleDelete={deleteEntity}
```

Comment:
Würde hier onEdit hier auch besser finden...

Reply:
handleDelete löscht wirklich. onSelect selectiert eigentlich nur eine entität.

Reply:
is jetzt onEdit